EIPS/eip-4519.md

```diff
@@ -2,9 +2,9 @@
 eip: 4519
 title: Non-Fungible Tokens Tied to Physical Assets (SmartNFT)
 description: Standard interface for non-fungible tokens representing physical assets that can generate or recover their own accounts and obey users.
```

Comment:
We now try to discourage "standard" in titles and description (since most EIPs are standards for something or other):

```suggestion
description: Interface for non-fungible tokens representing physical assets that can generate or recover their own accounts and obey users.
```